vmm/src/cpu.rs

```diff
@@ -1097,10 +1111,11 @@ impl CpuManager {
         Ok(())
     }
 
-    pub fn create_boot_vcpus(&mut self, entry_point: Option<EntryPoint>) -> Result<()> {
+    pub fn create_boot_vcpus(&mut self) -> Result<Vec<Arc<Mutex<Vcpu>>>> {
         trace_scoped!("create_boot_vcpus");
 
-        self.create_vcpus(self.boot_vcpus(), entry_point)
+        self.create_vcpus(self.boot_vcpus())?;
+        self.configure_vcpus(entry_point)
```

Comment:
@michael2012z This is the commit and the problematic line.

Reply:
I made a mistake in modifying the commit in middle. Thanks.
Now it's OK. Please take a look.